tests/test_all_plugins.py

```diff
@@ -16,34 +13,6 @@
     pytest.skip("skipping plugin specific tests", allow_module_level=True)
 
 
-@pytest.fixture(scope="session")
-def plugin_env():
-    try:
-        with isolated_plugin_env(PLUGIN) as env:
-            yield env
-    except CalledProcessError as e:
-        if "Failed building wheel" in str(e.output):
-            yield None
-
-
-def test_entry_points_importable(plugin_env):
```

Comment:
Can you give me a bit more context around this removal? Was it always xfail-ing?

Reply:
there is no CI job with `TEST_PACKAGE_NAME` defined so it land in xfail branch.